ca.hamaluik.Timecop.yml

```diff
@@ -20,6 +20,8 @@ modules:
       - mkdir /app/lib/
       - ln -s /app/timecop/timecop /app/bin/timecop
       - ln -s /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 /app/lib/libsqlite3.so
+      - if [ ! -e '/app/lib/libsqlite3.so' ]; then ln -s -f /usr/lib/aarch64-linux-gnu/libsqlite3.so.0 /app/lib/libsqlite3.so; fi
+      - if [ $(arch)="aarch64"  ]; then sed -si "/Exec=/cExec=env LIBGL_ALWAYS_SOFTWARE=1 timecop" timecop/ca.hamaluik.Timecop.desktop; fi
```

Comment:
this actually won't work because the .desktop file is NOT used when launching with "flatpak run". Also setting an environment isn't in the spec. If you want to set an environment variable you have to make a wrapper script. (or set it in the finish args, but it's not possible per arch, so I'm not suggesting it here)

Reply:
@hfiguiere I've never written an execution script before, though I have some experience with bash scripts. How should I refer to the executable in my script? Via an absolute path? Via a relative path that gets the file's current directory somehow (but not via "./", as the working directory can differ from the file's directory)?

Neither option seems very elegant to me, but I'm guessing one of those two is the recommended approach?